docker-compose.yml

```diff
@@ -69,5 +80,21 @@ services:
     volumes:
       - ./api/openapi:/usr/share/nginx/html/doc
 
+
+  # docker compose always mount named volumes as root (https://github.com/docker/compose/issues/3270)
+  # and without changing permission, we want be able to access shared volume
+  shared-volume-init:
+    image: ubuntu
+    user: root
+    group_add:
+      - '1000'
+    volumes:
+      - shared-volume:/tmp/2fas
+    command: chown -R 1000:1000 /tmp/2fas
+
+
 volumes:
   go-modules:
```

Comment:
TBH it doesn't look as valid yaml. But if docker doesn't complain, I won't object it as well.